docs/ht_ww.md

```diff
@@ -1,9 +1,53 @@
 # OVOS Listener - WakeWords / HotWords
-OVOS uses "wakewords" to activate the system.  This is what "hey Google" or "Alexa" is on proprietary devices.  By default, OVOS uses the WakeWord "hey Mycroft".
+OVOS uses "wakewords" to activate the system. This is what "hey Google" or "Alexa" is on proprietary devices. By default, OVOS uses the WakeWord "hey Mycroft".
+
+OVOS "hotwords" is the configuration section in `mycroft.conf` config  to specify what the WakeWord do.  Multiple "hotwords" can be used to do a variety of things from putting OVOS into active listening mode, a WakeWord like "hey Mycroft", to issuing a command such as "stop" or "wake up"
+
+Each hotword can do one or more of the following (with example configs):
+* trigger listening, also called a wake_word (default option)
+  * `“Listen” : “True”`
+
+* play a sound (only or with other action), you can set a specific sound per hotword.
+  * `"sound": "snd/start_listening.wav"`
+
+* emit a utterance to open a skill directly (instead of listen), useful to open skills from others.
+  * `“utterance” : “What’s the time”` 
+  * `“listen” : “false”`
+   
+  This example will start the date & time skill with the given utterance and not listen for a command.
+
+* emit a bus event directly to the messagebus, to for instance open a skill you developed yourself  (instead of listen / utterance), emit a message directly (microphone mute/unmute"
+  *	`“bus_event” : “custom message that you use in your own skill”`
+    
+     or
+   	
+  *	`“bus_event” : “mycroft.mic.mute"`
+
+* Use a hotkey; allows you to assign a keyboard shortcut (e.g., "ctrl+alt+h") for triggering the hotword manually. The ovos_ww_hotkeys module is         required forusing a hotkey.
+  *	`“Hotkey”: “Control+shift+h”`
+
+* assign a language for STT per wake word
+  * `Example?`
+    
+* be a wakeup word (out of sleep mode); take ovos-core out of sleep mode, also called a wakeup_word or standup_word
+  * Example:
+```
+  "listener": {
+    // Default wake_word and stand_up_word will be automatically set to active
+    // unless explicitly disabled under "hotwords" section with "active": false. This is usually not
+       desired unless you are looking to completely disable wake word usage
+    "wake_word": "hey mycroft",
+    "stand_up_word": "wake up"
```

Comment:
Looking for @JarbasAl confirmation that this syntax works as a wake word

Reply:
I agree, Jarbas should review this one because of what it is.

Reply:
under the "listener" section you specify the default wake_word and stand_up_word

these will be automatically set active: true (unless you override it in the individual config of each word)

in single wake word mode you never set active flag in each definition, you only change this value. when using a backend for example this is the only thing that changes

Reply:
This is a part I don't understand tbh, the difference between wake(up?)word and stand_up word as a hotword config.
So, I don't think my interpretation is helpful here, sorry.  I've copied the comments text from the technical manual or the dinkum listener (I've I recall correctly).

Reply:
wake_word -> start listening, "hey mycroft"
stand_up_word -> come out of sleep mode , default to "wake up"